docs/src/navigation/breadcrumbs/breadcrumbsSampleSeparatorString.jsx

```diff
@@ -40,13 +40,13 @@ const router = { // react-router core instance mockup
     ],
 };
 
-function BreadcrumbsSampleSeparatorIcon() {
+function BreadcrumbsSampleSeparatorString() {
     return (
         <Breadcrumbs
             router={router}
-            separatorIconType="caret-right"
+            separatorString=">"
```

Comment:
Let's use the name `dividerString` instead.

Reply:
done